src/main/java/com/cleanroommc/groovyscript/compat/mods/BasicGroovyPropertyContainer.java

```diff
@@ -0,0 +1,66 @@
+package com.cleanroommc.groovyscript.compat.mods;
+
+import com.cleanroommc.groovyscript.helper.ingredient.ItemsIngredient;
+import com.google.common.collect.AbstractIterator;
+import net.minecraft.creativetab.CreativeTabs;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.NonNullList;
+import net.minecraftforge.fml.common.Loader;
+import net.minecraftforge.fml.common.ModContainer;
+import net.minecraftforge.fml.common.registry.ForgeRegistries;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class BasicGroovyPropertyContainer {
```

Comment:
`BasicGroovyPropertyContainer` isnt a great name. `Property` isnt even relevant here. `ModWrapper` perhaps? would suggest `ModContainer` except for the obvious.